automated install/basic-install.sh

```diff
@@ -89,7 +89,9 @@ fi
 
 adlistFile="/etc/pihole/adlists.list"
 # Pi-hole needs an IP address; to begin, these variables are empty since we don't know what the IP is until this script can run
+# shellcheck disable=SC2269
```

Comment:
Please keep the PR focused on one topic. You can use another PR to add lint fixes for code that isn't touched by this PR.